en/faq/netlify-deployment.md

```diff
@@ -35,15 +35,24 @@ Press the _"New site from Git"_ button on the Netlify dashboard. Authenticate wi
 1. __Build command:__ `npm run build`
 1. __Publish directory:__ `dist`
 
-For a single page app there is a problem with refresh as by default on netlify the site redirects to *"404 not found"* this can be [prevented](https://www.netlify.com/docs/redirects/#rewrites-and-proxying) by configuring redirects. There is a module for that [nuxt-netlify](https://www.bazzite.com/docs/nuxt-netlify) which helps to configure both headers and redirects of the SPA .
+For a single page app there is a problem with refresh as by default on netlify the site redirects to *"404 not found"*. For any pages that are not generated they will fallback to SPA mode and then if you refresh or share that link you will get Netlify's 404 page.
 
-Alternativaly you can create a `_redirects` file (no file extension just _redirects) in the `static` folder with the following rules:
+The easiest way to fix this is by adding a generate property in your `nuxt.config` and setting `fallback: true`. Then it will fallback to the 200.html when in SPA mode.
 
 ```js
-/*    /200.html   200
-/*    /404.html   404
+export default {
+  generate: {
+    fallback: false
+  }
+}
 ```
 
+If however you want to automatically apply headers and redirects of the SPA then there is a module for that:
```

Comment:
Could you emphasize that the module is especially useful when you have *custom* headers/redirects (in a `_headers` or`_redirects` file)? :)

Reply:
done